modules/attachment/outputs.tf

```diff
@@ -7,3 +7,13 @@ output "attachment_parameters" {
   description = "SCC profile attachment parameters"
   value       = resource.ibm_scc_profile_attachment.scc_profile_attachment.attachment_parameters
 }
+
+output "sorted_list" {
+  description = "SCC profile attachment parameters"
+  value       = local.sorted_list
+}
+
+output "latest_profile" {
+  description = "SCC profile attachment parameters"
+  value       = local.latest_profile
+}
```

Comment:
I don't think either of these are needed as outputs. All a user will care about is maybe the profile name and version used by the attachment